src/tests/transfer.test.ts

```diff
@@ -149,13 +214,87 @@ describe('Transfer Action', () => {
             ta.transfer({
               fromChain: 'baseSepolia' as any,
               toAddress: receiver.address,
-              amount: '0.001',
+              amount: parseEther('0.0001'), // Very small amount
+              recipientAddress: receiver.address,
+              token: 'ETH',
+              tokenDecimals: 18,
+              data: '0x',
             })
           ).rejects.toThrow('Transfer failed');
         }
       });
     });
 
+    it('should work with Base Sepolia testnet (for ERC20 - USDC)', async () => {
+      const balance = await wp.getWalletBalanceForERC20('baseSepolia', 6, usdcAddressBase);
+      console.log(`Base Sepolia balance: ${balance} USDC`);
+
+      if (balance && parseFloat(balance) > 0.001) {
+        const result = await ta.transfer({
+          fromChain: 'baseSepolia' as any,
+          toAddress: usdcAddressBase,
+          amount: 1000000n,
+          recipientAddress: receiver.address,
+          token: 'USDC',
+          tokenDecimals: 6,
+          data: (await getTransferCallData(1000000n, receiver.address)) as Hex,
+        }); // Don't forget to change the address later!
+
+        expect(result.hash).toMatch(/^0x[a-fA-F0-9]{64}$/);
+        expect(result.to).toBe(usdcAddressBase);
+        expect(result.data.startsWith('0x')).toBe(true);
```

Comment:
Test that the encoded ABI is actually correct instead of just having it be `0x`.